src/components/CardActions.tsx

```diff
@@ -0,0 +1,77 @@
+import { CardAttributes } from "@/types/interfaces";
+
+interface CardActionsProps {
+  card: CardAttributes;
+}
+
+export default function CardActions({ card }: CardActionsProps) {
+  const handleCopyRawText = () => {
+    navigator.clipboard.writeText(card.description_raw).then(() => {
+      alert("Raw text copied to clipboard!");
+    });
+  };
+
+  const handleCopyRawJson = () => {
+    navigator.clipboard.writeText(JSON.stringify(card, null, 2)).then(() => {
+      alert("Raw JSON copied to clipboard!");
+    });
+  };
+
+  const handleViewCard = () => {
+    window.open(`https://lor.gg/card/${card.card_code}`, "_blank");
+  };
+
+  const handleDownloadImage = () => {
+    const link = document.createElement("a");
+    link.href = card.assets[0].game_absolute_path;
+    link.download = `${card.name}.png`;
+    link.click();
+  };
+
+  const handleDownloadFullArt = () => {
+    const link = document.createElement("a");
+    link.href = card.assets[0].full_absolute_path;
+    link.download = `${card.name}_full.png`;
+    link.click();
+  };
+
+  return (
```

Comment:
Not a comment to change but just a curiosity, since all of your className fields for you buttons are the same do you typically abstract that code somewhere else or just leave as is?

Reply:
Depends on the project for me, normally i like to keep them as is incase I have to make an adjustment to 1 and not the others later, plus its easier (in this case) than making a button component for what is essentially a broader button component